discovery_engine_core/core/src/engine.rs

```diff
@@ -1759,9 +1763,7 @@ pub(crate) mod tests {
                 .set_body_string(include_str!("../test-fixtures/newscatcher/duplicates.json"));
 
             Mock::given(method("POST"))
-                .and(path_regex(
-                    "/newscatcher/headlines-endpoint-name|/newscatcher/v2/trusted-sources|newscatcher/news-endpoint-name",
-                ))
```

Comment:
❔ could you remind me what was the reason for this regex?

Reply:
we used routes in the tests which return a mocked fixed result. at some point additional routes were used in the tests, but they don't actually matter since we always return the same mocked result anyways. hence i simplified this to avoid to update outdated routes, resp. to expose the internal routes from the providers crate.